net/cloudflared/files/cloudflared.config

```diff
@@ -4,10 +4,14 @@ config cloudflared 'config'
 	option token ''
 	option config '/etc/cloudflared/config.yml'
 	option origincert '/etc/cloudflared/cert.pem'
-	option edge-bind-address 'auto'
-	option protocol 'auto'
+	option edge_bind_address ''
+	option edge_ip_version ''
+	option grace_period ''
+	option protocol ''
```

Comment:
should be `http2` for now

Reply:
Should we really set `http2` by default? At least for me the default behavior, which uses `quic` first, fallback to `http2` if failed, works well, and most users should benefit from this. Some users may need to set `http2` explicitly as their ISP is not quic-friendly, reported in this [forum post](https://community.cloudflare.com/t/cloudflared-keeps-falling-back-from-quic-to-http2/385031).

Reply:
Github reviewing is nice...

Reply:
Nah, `quic` requires cloudflare patched go toolchain, which is not supported here.
Only `http2` is functional currently, so this is the reason that PR was created.

Reply:
Copy sir